Insurance data set (#62)
deephaven/examples: Insurance/README.md

# Insurance dataset

This folder contains one sub-folder, `csv`, which contains the `insurance.csv` file.  This file contains health indicators of patients and their insurance charges.  These can be used to predict insurance charges given their own health indicators.

## Fields in the file

`Age` - Age of the patient in years
`Sex` - Biological sex of the patient
`BMI` - The patient's BMI
`Children` - The number of children the patient has
`Smoker` - `Yes` or `No` if the patient smokes cigarettes or not
`Region` - The region of the United States the patient resides in

> `Expenses` - The patient's insurance expenses# Insurance dataset

Reviewer: ```suggestion
`Expenses` - The patient's insurance expenses
```

> This folder contains one sub-folder, `csv`, which contains the `insurance.csv` file.  This file contains health indicators of patients and their insurance charges.  These can be used to predict insurance charges given their own health indicators.

Reviewer: accidentally duplicated?

# Insurance dataset

> This folder contains one sub-folder, `csv`, which contains the `insurance.csv` file.  This file contains health indicators of patients and their insurance charges.  These can be used to predict insurance charges given their own health indicators.

Reviewer 2: ```suggestion
This folder contains one sub-folder, `csv`, which contains the `insurance.csv` file.  This file contains health indicators of patients and their insurance charges.  These can be used to predict insurance charges based upon health indicators.
```

## Fields in the file

`Age` - Age of the patient in years
`Sex` - Biological sex of the patient
`BMI` - The patient's BMI
`Children` - The number of children the patient has
`Smoker` - `Yes` or `No` if the patient smokes cigarettes or not
`Region` - The region of the United States the patient resides in
`Expenses` - The patient's insurance expenses# Insurance dataset

## Source and license

> This data was built from [Kaggle](https://kaggle.com).  The original link, authored by Brandon Yong, can be found [here](https://www.kaggle.com/brandonyongys/insurance-charges/notebook#Basic-data-information).  It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.

Reviewer 2: ```suggestion
This data was obtained from [Kaggle](https://kaggle.com).  The original analysis and data, authored by Brandon Yong, can be found [here](https://www.kaggle.com/brandonyongys/insurance-charges/notebook#Basic-data-information).  This data is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.
```
